challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js

```diff
@@ -0,0 +1,17 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+/* RESPONSE */
+
+// for even numbers
+for (let number = 10; number <= 40; number += 2) {
```

Comment:
These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the even ones